configserver/src/main/java/com/yahoo/vespa/config/server/deploy/ModelContextImpl.java

```diff
@@ -217,8 +214,9 @@ public String athenzDnsSuffix() {
         @Override
         public double defaultTermwiseLimit() { return defaultTermwiseLimit; }
 
+        // TODO: Remove
         @Override
-        public boolean useBucketSpaceMetric() { return useBucketSpaceMetric; }
+        public boolean useBucketSpaceMetric() { return true; }
```

Comment:
Consider making a default implementation of the method in the interface, then you could remove this now

Reply:
I don't think I can do that since the interface is in config-model-api: Old deployments would only see the old interface and therefore no default implementation.

Reply:
Maybe I don't get this, but old config models implement the method, so that should not be an issue. Anyway, this is nitpicking, so I wouldn't bother.

Reply:
The implementation is in ModelContextImpl, in the configserver bundle, and is always at the latest config server Vespa version, and independent on the particular version of the model running?

Reply:
You are completely right, I wrongly assumed this was in the model. Thanks.